### PR TITLE
Incorporate logger and IPFS client

### LINKED ISSUE #930: [Core] harmonize naming of package imports
**Problem:** There's a lack of consistency in how imported packages are (short)named: e.g. `chns "github.com/eris-ltd/eris-cli/chains"`, among others. The short-hand for `. "github.com/eris-ltd/common"` is fine _if you already know the code_ but doesn't make it easy for new contributors to navigate.

**Solution:** harmonize

### LINKED ISSUE #1006: [Logging] Revert eris-logger to utilize a common log package hosted in cli repository
All the tools should run from here.

### PR DESCRIPTION
**Pull request by pietv**
* Move `eris-logger` code to `log`. Closes #1006
* Move IPFS client code to `util/ipfs.go` (Other `common` code is pending resolution of import cycles).
* Harmonization of import paths and file names. Closes #930, #946.

### CODE REVIEW

File: chains/chains.go [hunk 1]
```diff
@@ -411,3 +578,25 @@ func whatChainStuffExists(chainName string) (bool, bool, bool, bool) {
 
 	return chainDirExists, chainConfigExists, chainDataExists, chainContainerExists
 }
+
+func exportFile(chainName string) (string, error) {
+	fileName := util.GetFileByNameAndType("chains", chainName)
+
+	return ipfs.SendToIPFS(fileName, "", "")
+}
+
```
Comment (zramsay) [inline chains/chains.go]: this function is latent code and can be removed

File: chains/chains.go [hunk 2]
```diff
@@ -411,3 +578,25 @@ func whatChainStuffExists(chainName string) (bool, bool, bool, bool) {
...
+func checkKeysRunningOrStart() error {
```
Comment (zramsay) [inline chains/chains.go]: there's a generalized function `services.EnsureRunning()`  that we could use instead

File: chains/chains.go (hunk 1 above)
Comment (pietv) [inline chains/chains.go]: Yep. Good catch. :+1: I have a list of changes I'd like to make to an IPFS client, so I'd prefer to have it as a separate ticket. The idea for this one is just to migrate _as is_.

**Comment by zramsay** [comment]
LGTM :) so much cleaner. The code is much more approachable too.

**Pull request merged**